### ISSUE TITLE
There should be a way to declare dependency between quickstarts.

### ISSUE THREAD

**Issue by wtrocki** [cloudmosaic/quickstarts#30]
We have number of quickstarts that depend on some others.

For example user can create kafka using cli OR UI. Then having kafka is prerequisite of quarkus guide etc.

**Comment by wtrocki** [comment]
CC @pmuir @jschuler

**Comment by DuncanDoyle** [comment]
My current requirement is that I have 2 Quick Starts, one that explains how to connect your Service instance to OpenShift and one that explains how to use Service Binding to bind your Quarkus application to the service. I would like to be able to have a link at the end of the first Quick Start that opens the next Quick Start.

**Comment by pmuir** [comment]
This already implemented via `nextQuickStart` see https://github.com/bf2fc6cc711aee1a0c2a/guides/blob/main/getting-started/quickstart.yml#L21-L23

**Comment by wtrocki** [comment]
This is to mention couple of things as the next steps, but it will be also good to have the opposite direction.
People landing on guide they wanted to land - seeing what they can do else before doing it.

Some flow to visualize this

![Screenshot 2021-03-22 at 16 55 34](https://user-images.githubusercontent.com/981838/112027783-70f8a800-8b2f-11eb-8682-69fa41c5b882.png)

**Comment by pmuir** [comment]
These would be the "prerequisites". @jschuler I'm not sure if we have the "magic linking" for the prereqs?

**Comment by wtrocki** [comment]
Yes.. if results of this issue would be - "Please put link to quickstart in prerequisites" then I'm fine :D

**Comment by pmuir** [comment]
Yes, you should put it there. But we want to allow you to do it by ID, rather than by url.

**Comment by jschuler** [comment]
That would be a nice feature to add. I've seen prerequisites in OpenShift that reference another quick start, so this seems like it could be useful. Here's an example from OS:
<img width="639" alt="Screen Shot 2021-03-22 at 6 22 35 PM" src="https://user-images.githubusercontent.com/869106/112065901-d8b4f000-8b3b-11eb-9581-bf9cf584c08d.png">
Today there is no special parsing of the prerequisites, it's just a string array.
